tests/config/pytest_filter.yml

```diff
@@ -72,7 +72,7 @@ wf_theiacov_ont:
   - tasks/taxon_id/task_nextclade.wdl
   - tasks/species_typing/betacoronavirus/task_pangolin.wdl
   - tasks/species_typing/lentivirus/task_quasitools.wdl
-  - tasks/quality_control/basic_statistics/task_sc2_gene_coverage.wdl
+  - tasks/quality_control/basic_statistics/task_gene_coverage.wdl

```

Comment:
Same comment x3 as above RE cross-talk

Reply:
yikes, yes it is lol 😂

Reply:
should we keep or remove?

Reply:
Haha let's remove it. It will be a confusing conflict for other branches that may get PR'ed before #341

Reply:
removed!